File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/user_statistics_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class UserStatisticsVO:
    """用户统计响应"""
    created_at: str = None
    updated_at: str = None
    user_id: str = None
    username: str = None
    avatar: str = None
    registered_at: str = None
    last_login_at: str = None
    login_count: int = None
    total_usage_time: int = None
    total_generations: int = None
    total_projects: int = None
    total_files: int = None
    total_storage: int = None
    level: UserLevelVO = None
    vip_info: VipStatisticsVO = None
    achievement_stats: AchievementStatisticsVO = None
    usage_stats: UsageStatisticsVO = None
    generation_stats: GenerationStatisticsVO = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/UserStatisticsVO.kt
package com.sdkwork.app

data class UserStatisticsVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val userId: String? = null,
    val username: String? = null,
    val avatar: String? = null,
    val registeredAt: String? = null,
    val lastLoginAt: String? = null,
    val loginCount: Int? = null,
    val totalUsageTime: Int? = null,
    val totalGenerations: Int? = null,
    val totalProjects: Int? = null,
    val totalFiles: Int? = null,
    val totalStorage: Int? = null,
    val level: UserLevelVO? = null,
    val vipInfo: VipStatisticsVO? = null,
    val achievementStats: AchievementStatisticsVO? = null,
    val usageStats: UsageStatisticsVO? = null,
    val generationStats: GenerationStatisticsVO? = null
)
